src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int LINE_WIDTH = 60;
+
+    //to draw a straight line
+    private static void printHorizontalLine() {
+        System.out.println("_".repeat(LINE_WIDTH));
+    }
+
+    public static String[] taskLister(String[] todos) {
+
+        int todoCount = 0;
+        String[] result = new String[100];
+        for (String todo : todos) {
+            //System.out.println(word);
+            if (todo != null) {
+                result[todoCount] = todo;
+                todoCount++;
+            } else {
+                break;
+
+            }
+        }
+        return Arrays.copyOf(result, todoCount);
+    }
+
+    public static boolean splitString(String line) {
+        String[] words = line.split(" ");
+        for (String word : words) {
+            if (word.equals("done")) {
+                {
+
+                    return words[1].matches(".*\\d.*");
+
+                }
+
+            } else return false;
+        }
+
+        return false;
+    }
+
+
+    public static String[] checkMarker(String[] tobeCheckeds, int doneNumber, int listedcount) {
+        String[] checkedMarks = new String[tobeCheckeds.length];
+
+        for (int i =0; i<listedcount; i++ ) {
+            if (tobeCheckeds[i]== null ) { //&& !tobeCheckeds[i].equals("X")
+                checkedMarks[i] = " ";
+            } else
+                {
+                    checkedMarks[i] = tobeCheckeds[i];
+                }
+            }
+
+        checkedMarks[doneNumber] = "X";
+        return checkedMarks;
+    }
+
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        //Level0
+        printHorizontalLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println();
+        printHorizontalLine();
+        String line = "";
+        Scanner in = new Scanner(System.in);
+
+        //initialise the 100 todos items array
+        String[] todos = new String[100];
```

Comment:
Good job in using the plural form to represent a collection of objects! But to do doesn't really have a plural so maybe can rename?